php-classes/Slate/CBL/Competency.php

```diff
@@ -271,8 +288,9 @@ public function getCompletionForStudent(Student $Student, $level = null, $demons
         return [
             'demonstrationsLogged' => 0,
             'demonstrationsComplete' => 0,
+            'baselineRating' => $StudentCompetency && $StudentCompetency->BaselineRating ? floatval($StudentCompetency->BaselineRating) : null,
             'demonstrationsAverage' => null,
-            'currentLevel' => $currentLevel
+            'currentLevel' => null
```

Comment:
Oh, yeah. As long as the new method doesn't have this regression... This whole thing can just get deleted